### PR TITLE
Add cognitive service AnomalyFinder

### PR DESCRIPTION
**Pull request by moreOver0**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json
Comment (yangyuan) [inline specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json]: operationId should be {OperationGroupName}_{OperationName}

in SDK, this API will be mapped to
`AnomalyFinderClient.{OperationGroupName}Operations.{OperationName}()`

So we need to be careful about these names.

**Review by yangyuan — CHANGES_REQUESTED** [review]

File: specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json
Comment (moreOver0) [reply specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json]: fixed.

File: specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json
Comment (moreOver0) [reply specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json]: done

**Comment by jhendrixMSFT** [comment]
Looks good from SDK perspective, once @yangyuan approves I can merge.

**Comment by yangyuan** [comment]
Hi @jhendrixMSFT, we are waiting for API review board's review.
Once we get approved I will let you know, thanks !

**Comment by jhendrixMSFT** [comment]
@yangyuan hello any update?

**Comment by yangyuan** [comment]
@jhendrixMSFT I don't think AnomalyFinder has finished the reviewing.
@moreOver0 whats the current review status?

File: specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json
Comment (johanste) [inline specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json]: Please allow/follow [ISO8601/ECMA script formats](https://www.ecma-international.org/ecma-262/5.1/#sec-15.9.1.15).

File: specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json
Comment (johanste) [inline specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json]: I would suggest merging isAnomaly, isNegativeAnomaly and isPositiveAnomaly into a single array of numbers. This reduces the amount of network traffic/time to serialize/deserialize.

File: specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json
Comment (johanste) [inline specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json]: Is this really a float? Please verify that you are using the correct number type (here and in other places)

File: specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json
Comment (conhua) [reply specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json]: will take it

File: specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json
Comment (conhua) [reply specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json]: It should be an integer

File: specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json
Comment (conhua) [reply specification/cognitiveservices/data-plane/AnomalyFinder/stable/v2.0/AnomalyFinder.json]: After discussing with team members, we would not merge the three arrays into one. The reasons are:  

1. Based on feedback of existing customers,  some users needs to distinguish anomalies as positive ones or negative ones,  others only use isAnomaly as their final results.  If we combine these together, they have to understand these concept.
2. From Maintenance aspect,  we may deprecate positive and negative concept or replace it as spike or dips, or even add change point. In separate ways, we can remove or add arrays.
3. Positive and negative are additional features we provided on top of anomalies,  if we merge them together,  an incorrect judgement on positive or negative direction may make customer confuses whether it's a true anomaly or not.

**Comment by moreOver0** [comment]
@yangyuan Could you approve this PR since we did't hear further feedback from review board by last Friday as we negotiate in email?

**Review by yangyuan — APPROVED** [review]

**Comment by jhendrixMSFT** [comment]
Closing then re-open to fix CI.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**
